lib/rules/disallow-multiple-var-decl.js

```diff
@@ -58,15 +69,26 @@ module.exports = function() {};
 module.exports.prototype = {
 
     configure: function(options) {
-        assert(
-            options === true ||
-            options === 'strict' ||
-            options === 'exceptUndefined',
-            this.getOptionName() + ' option requires a true value, "strict", or "exceptUndefined"'
-        );
-
-        this.strictMode = options === 'strict';
-        this.exceptUndefined = options === 'exceptUndefined';
+        if (typeof options === 'object') {
+            if (Array.isArray(options.allExcept)) {
+                this.exceptUndefined = options.allExcept.indexOf('undefined') > -1;
+                this.exceptRequire = options.allExcept.indexOf('require') > -1;
+            }
+
+            this.strictMode = options.strict === true;
+        } else {
+            assert(
+                options === true ||
+                options === 'strict' ||
+                options === 'exceptUndefined' ||
+                options === 'exceptRequire',
```

Comment:
I'd prefer if this new option was not supported. If users want to exclude `require`s, then they should use the `allExcept` form (which would also help them migrate to 2.0).